doc/apple2enh.sgml

```diff
@@ -472,6 +472,7 @@ BASIC.SYSTEM) there are some limitations for DOS 3.3:
 
 The enhanced&nbsp;Apple&nbsp;//e has no color text mode. Therefore the functions
 <tt/textcolor()/, <tt/bgcolor()/ and <tt/bordercolor()/ have no effect.
+The driver checks the Apple&nbsp;//c 80/40 switch on program startup.
```

Comment:
I think the term 'driver' doesn't apply well to CONIO.

Reply:
I suggest
```
<tt/cputc()/ checks the Apple&nbsp;//c 80/40 switch on program startup.
```